src/plugins/gui/providers/banxaProvider.ts

```diff
@@ -182,6 +184,16 @@ export const banxaProvider: FiatProviderFactory = {
       partnerIcon,
       pluginDisplayName,
       getSupportedAssets: async (paymentTypes: FiatPaymentType[]): Promise<FiatProviderAssetMap> => {
+        // Return nothing if paymentTypes are not supported by this provider
+        let foundPaymentType = false
+        for (const type of paymentTypes) {
+          const t = asFiatPaymentType(type)
+          if (allowedPaymentTypes[t]) {
+            foundPaymentType = true
+          }
+        }
+        if (!foundPaymentType) return { crypto: {}, fiat: {} }
```

Comment:
Same comments as previous commit.

We could probably squash all of these commits?